lib/SparkFun_APDS-9960_Sensor_Arduino_Library/src/SparkFun_APDS9960.cpp

```diff
@@ -65,20 +65,20 @@ SparkFun_APDS9960::~SparkFun_APDS9960()
  */
 bool SparkFun_APDS9960::init()
 {
-    return init(DEFAULT_GGAIN, DEFAULT_GLDRIVE, DEFAULT_AGAIN, DEFAULT_PGAIN, DEFAULT_LDRIVE);
+    return init(DEFAULT_GGAIN, DEFAULT_GLDRIVE, DEFAULT_PGAIN, DEFAULT_AGAIN, DEFAULT_LDRIVE);

```

Comment:
Is there a reason these are now swapped in the library?
If it is to make it work again with our implementation, please change the call to this function.
It looks like this is an external library and if we at one time like to upgrade to the latest of that library (or someone uses the ArduinoIDE to install the libraries) we will run into issues which are not noticed by the compiler.

Reply:
a) I made a booboo adding the arguments to that init method, that's what I fixed here.
b) The library was already modified by the original author of the plugin, I built on that, as the source is included in the ESPEasy repo
c) The update-rate/PR-acceptance is 'rather slow' on the original library, and it doesn't support settable parameters anywhere, except for enabling interrupts, and I had to add these parameters somewhere, co I chose this method. Could just as well have been an extra constructor.
If you want that changed, I can do that, but this is what I tested so far.

Reply:
Well the main argument is to keep it in sync with the "original" library.

Reply:
I'd like that, but it is impossible, as the required 'features' aren't available from the original library.